Wangscape/noise/module/ModuleFactories.cpp

```diff
@@ -0,0 +1,109 @@
+#include "ModuleFactories.h"
+#include "Gradient.h"
+#include "NormLPQ.h"
+#include "../MakeReseedable.h"
```

Comment:
Could you use full paths in include? It helps to determine, which file do you include (e.g if you add `Wangscape/Gradient.h` and remove `Wangscape/noise/module/Gradient.h` it could still work (as root dir is added to include directories in `CMakeLists.txt`).